example/src/modules/ImageCrush.js

```diff
@@ -42,9 +42,13 @@ const ImageCrush = ({
     const imageUrl = `${baseServiceUrl}?width=${_width}&height=${_height}&original_uri=${encodeURIComponent(
       url
     )}`;
-    const key = getKey();
+    const key = getKey() ? getKey() : process.env.NODE_ENV === "development" ? 'test-key' : '';
     let headers = {};
 
+    if (process.env.NODE_ENV === "development" && key === '') {
```

Comment:
this condition will always fail since `key` will always have a value in development env. Use:
```
f (process.env.NODE_ENV === "development" && !getKey()) {
```